packages/v4/src/content/get-started/design.md

```diff
@@ -7,24 +7,33 @@ import './get-started.css';
 
 Whether you are converting from a PatternFly 3 application or starting fresh with the latest and greatest PatternFly design system, this document is intended to help provide designers with information regarding resources and support available.
 
-
 ## Getting the design kit
 
-### Before you begin
-The design kit is a collection of Sketch assets that make it easy for designers to create hi-fi mockups that accurately represent PatternFly components and layouts. It works with Sketch libraries.
+The PatternFly design kit was originally developed for [Sketch users](#Mac-users), but is now available for use through [any design application](#Non-mac-users).
+
+### Mac users
+
+#### Before you begin
+The design kit is a collection of Sketch assets that make it easy for designers to create hi-fi mockups that accurately represent PatternFly components and layouts. It works with Sketch libraries.To use it, you need to install [Sketch version 68](https://www.sketch.com/) or above. 
+
+#### Download the font
+1. PatternFly currently uses Red Hat Text and Red Hat Display. To download these fonts, go to https://github.com/RedHatOfficial/RedHatFont and download the whole repository. Then select all files in the "TTF" folder and double-click. This will install them onto your computer locally.
+
+#### Download the template file
+2. Access the [PatternFly template file](https://www.sketch.com/s/729c2eee-e8b6-4fcd-8a79-f6faa8c30f89) on Sketch cloud.
+3. Click Download Document to open the file. Then, click File → Save as template.
```

Comment:
I feel like the buttons to click on should be italicized or something to differentiate them from the rest of the text. So in this case "Download document"